practice-app/practice-app/website/api/equipment.py

```diff
@@ -0,0 +1,48 @@
+from flask import Blueprint, render_template, request, flash, json, redirect, url_for
+from flask_login import login_required, current_user
+from ..models import Equipment
+from .. import db
+import requests
+import json
+
+equipments = Blueprint('equipments', __name__)
+
+@equipments.route('/', methods=['GET', 'POST'])
+@login_required
+def post_equipment():
+    if request.method == 'POST':
+
+        if not request.json or not 'name' in request.json:
+            return "Parameter not correct", 400
+
+        req = requests.get('https://api.blockchain.com/v3/exchange/tickers')
+        result = req.json()
+
+        print(result)
+
+        data = json.loads(req.content)
+
+        price = data[0]['price_24h']
+
+        enteredEquipment = request.form.get('equipment')
+
+        tempEquipment = Equipment.query.filter_by(equipmentName=enteredEquipment).first()
+        if tempEquipment:
+            flash('Equipment already exists!.', category='error')
+        else:
+            new_equipment = Equipment(equipmentName=enteredEquipment, price=price)
+            
+            try:
+                db.session.add(new_equipment)
+                db.session.commit()
+                flash('Successfull', category='success')

```

Comment:
flash is related to frontend, maybe returning with correct code would be better here